cardano-node-emulator/test/Cardano/Node/Emulator/MTLSpec.hs

```diff
@@ -59,6 +61,19 @@ tests =
             void $ payToAddress (w1, pk1) w2 (Value.adaValueOf 1)
             nextSlot
         )
+    , checkPredicateOptions options "submit staking tx" (hasValidatedTransactionCountOfTotal 1 1) $ do
+        let
+          Right stakeKeyHash = toCardanoStakeKeyHash . unPaymentPubKeyHash . paymentPubKeyHash $ knownMockWallet 1
+          stakeCred = C.StakeCredentialByKey stakeKeyHash
+          stakeCert =
+            C.makeStakeAddressRegistrationCertificate
+              (C.StakeAddrRegistrationPreConway C.ShelleyToBabbageEraBabbage stakeCred)
+          tx =
+            E.emptyTxBodyContent
+              { C.txCertificates = C.TxCertificates C.shelleyBasedEra [stakeCert] (C.BuildTxWith mempty)
```

Comment:
@james-iohk It feels weird that this validates: an empty BuildTxWith and no extra witnesses needed?